### PR TITLE
fix: display starkName

### LINKED ISSUE #24: Make the connection button resolve stark domains and PFPs
## Description 📹

Our top right-corner button is not showing .stark domains and PFPs. We need to make it so so it looks like so

Capture d’écran 2024-11-15 à 09 55 13

## Proposed Actions 🛠️

Here’s a checklist of actions to follow for resolving this issue:

1. **Fork and Create Branch**:  
   Fork the repository and create a new branch using the issue number:
   ```bash
   git checkout -b fix-[issue-number]
   ```

2. **Implement Changes**:  
  - [ ] Refactor the button component to make it resolve stark domains and PFPs using this [button component](https://github.com/lfglabs-dev/app.starknet.id/blob/testnet/components/UI/connectButton.tsx) as model
  - [ ] Make it work taking example from  in [this context](https://github.com/lfglabs-dev/app.starknet.id/blob/25f533b385da3e1a4024c24643119802ef1a2e24/components/UI/navbar.tsx#L171)

3. **Run Tests and Commit Changes**:  
   Make sure your changes don't break existing functionality and commit with a clear message:
   ```bash
   git commit -m "Fix: [Short description of the fix]"
   ```

## Required 📋

To keep our workflow smooth, please make sure you follow these guidelines:

- **Assignment**: Don't create a pull request if you weren’t assigned to this issue.
- **Timeframe**: Complete the task within **3 business days**.
- **Closing the Issue**: In your PR description, close the issue by writing `Close #[issue_id]`.
- **Review Process**:
  - Once you've submitted your PR, change the label to **"ready for review"**.
  - If changes are requested, address them and then update the label back to **"ready for review"** once done.
- **Testing**: Test your PR locally before pushing, and verify that tests and build are working after pushing.

Thank you for your contribution 🙏

⚠️ WARNING: Failure to follow the requirements above may result in being added to the OnlyDust blacklist, affecting your ability to receive future rewards.

### PR DESCRIPTION
**Pull request by NueloSE**
Closes #24

### CODE REVIEW

**Comment by NueloSE** [comment]
demo

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]
I don't see my pfp

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]
image

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]
You can check on our other repos to see how it works, e.g.: https://github.com/lfglabs-dev/app.starknet.id

**Comment by NueloSE** [comment]
Screenshot from 2024-12-19 17-00-35



ready for review @Marchand-Nicolas

File: hooks/useNotificationManager.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+const notificationsAtom = atomWithStorage<SIDNotification<NotificationData>[]>(
+  'userNotifications_SID',
+  []
+);
+
+export function useNotificationManager() {
+  const { provider } = useProvider();
+  const { address } = useAccount();
+  const [notifications, setNotifications] = useAtom(notificationsAtom);
```
Comment (Marchand-Nicolas) [inline hooks/useNotificationManager.ts]: Are you sure you need jotai and useAtom ? Can't you do the same with useState from React ?

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]

**Review by Marchand-Nicolas — APPROVED** [review]
Really good job lgtm!

**Pull request merged**